rediscluster/client.py

```diff
@@ -96,7 +96,7 @@ class StrictRedisCluster(StrictRedis):
     )
 
     def __init__(self, host=None, port=None, startup_nodes=None, max_connections=32, init_slot_cache=True,
-                 pipeline_use_threads=True, readonly_mode=False, reinitialize_steps=None, **kwargs):
```

Comment:
This change would require a note in the `docs/Upgrading.md` file stating that this argument can be removed from the implementers code.